_editorial_policies/02_editor_instructions.md

```diff
@@ -11,23 +11,26 @@ This provides instructions to editors handling manuscripts for LiveCoMS.
+Once the presubmission letter is approved, the author proceeds with article preparation and ultimately submit it to that section of LiveCoMS.
+The Lead Editor then checks that the article is consistent with what was proposed in the presubmission letter, and then assigns it to an appropriate Associate Editor who manages the review process and makes a final decision on the manuscript, reporting the final decision to the authors. Usually this Associate Editor will be the same one that handled the original presubmission letters.
+
+## Pre-submission letters
+
+Assuming the subject matter proposed in the is within the scope of the LiveCoMS and is otherwise acceptable, the Section Lead Editor assigns this presubmission letter to and Associate Editor. Reasons for the letter not being acceptable include ignoring key required elements of the presubmission letter, being too similar to existing LiveCoMS articles. The Section Lead Editor signs off on the final presubmission letter, and the Associate Editor notifies the author that the letter is approved.  Instrutions for the presubmission letter are [found here](https://github.com/livecomsjournal/livecomsjournal.github.io). 
 
 ## Pre-review processing
 
 ### Lead Editor
 
-The Lead Editor:
-- Handles analysis of presubmission inquiries
-- On article submission, confirms that the article is consistent with the presubmission letter
-- Passes the article and presubmission letter to an appropriate Associate Editor to manage the review process (unless he or she decides to manage the review process directly)
-- Makes a final decision after receiving a recommendation from the Associate Editor and reviewers (see below)
+The Section Lead Editor:
+- Passes the article and presubmission letter to an appropriate Associate Editor to manage the review process (unless he or she decides to manage the review process directly). Usually, this will be the Associate Editor who handled the presubmission letter.
+- Usually will delegate final decision to the Associate Editor. If the Section Lead Editor anticipates their input may be necessary, they will notify the Associate Editor at the time of the manuscript submission, or if not, as soon as they become aware additional input may be necessary. They may overrule the Associate Editor after acceptance, but this should only be done in exceptional cases, for example, fraud or plagarism. If there are important scientific changes that need to be made in the article that were not caught by Associate Editors previous to acceptance, these can be handled by an unofficial request for revision before the paper is officially posted, or between ASAP posting and the issue publication.  
```

Comment:
Perhaps note that the associate should get in touch with the section lead if there is any controversy expected (e.g. the reviewers disagree on the merits of the paper or some such).